### ISSUE TITLE
Special formatting for descriptions

### ISSUE THREAD

**Issue by KamasamaK** [foundeo/cfdocs#570]
It's clear that the backtick (`) character is used for special formatting of code, but this is not documented in the README. Are there any other special formatting options that should also be documented? I noticed that backticks were removed in commit d37ace4 and I think that it needs to be made clear that they are valid for formatting code.

**Comment by shaedrich** [comment]
Maybe something internal does not work with ecma6 template strings...

**Comment by KamasamaK** [comment]
@shaedrich I'm not understanding what you mean. Internal to what? I don't see what template strings have to do with JSON.

**Comment by pfreitag** [comment]
Yeah the ```  backtick wraps the value with a `<code></code>` tag. That should be added to the readme.

The only other tricks (that I can think of) are that if `CF[0-9.]+` or `Lucee[0-9.]+` are found it wraps the string with a bootstrap label.

**Comment by shaedrich** [comment]
There, <code>`</code> is used as quote. Not a good idea. We have single quotes, double quotes, masked quotes, both single and double. Optionally other signs like chevrons or guillemets would be possible.

With ECMA6, <code>`</code> are used to indicate [template strings](/lukehoban/es6features/blob/master/README.md#template-strings). I first thought, ECM6 is not supported because of whatever, but that is not the case.
